lib/containers/commands/tail.rb

```diff
@@ -6,6 +6,6 @@ class Containers::CLI < Thor
   method_option :service, type: :array, aliases: "-s", desc: "A list of service names (space delimited)"
   def tail(*args)
     args << "--since 5m" unless args.include?("--since")
-    execute_command "docker compose logs #{args.join " "} -f #{requested_service_names.join " "}"
+    execute_command "docker compose #{docker_compose_files.map { |f| "-f #{f}" }.join " "} logs #{args.join " "} -f #{requested_service_names.join " "}"
```

Comment:
This seems to be used a lot. I wonder if it should be added to a base class or concern?

```ruby
docker_compose_files.map { |f| "-f #{f}" }.join " "}
```

Reply:
Definitely could be refactored but I'll probably wait on that.